### ISSUE TITLE
Forestry Support Requested

### ISSUE THREAD

**Issue by ArsTechne** [ferreusveritas/Dynamic-Trees#203]
Requesting Forestry support be added. Running a family server and it's quite jarring to have a mix of vanilla like trees with Dynamic ones.

Yes, I did see there was another post in April 2018 on this topic, but it was closed because it "wasn't in the projects scope". This is preposterous as, like Thaumcraft, the Forestry mod has trees that could benefit this compatibility, but maybe that was at a different time of the project when the views were different. Or it could be the genetic aspect created issues. But it didnt help to give such a generic reasoning for the proposal to be turned down.

So I'm proposing the request for support again for the Forestry mod.
(And I understand It would be quite the undertaking as it has a lot of trees on top of the mod possibly having some oddities with them)

**Comment by Deshiba** [comment]
>  "wasn't in the projects scope". This is preposterous as, like Thaumcraft, the Forestry mod has trees that could benefit this compatibility, but maybe that was at a different time of the project when the views were different.

No, the Thaumcraft trees are added trough an add-on, it's not part of Dynamic trees itself. This will remain out of scope.

You are free to build an add-on or have it be built by someone capable trough good will or commission though.

**Comment by ghost** [comment]
You should have paid attention to the original before opening another issue for the same reason.  It's still out of project scope and always will be.  This feature will need to be provided as an add-on just like thaumcraft is and will not be done by myself or mangoose.

**Comment by ArsTechne** [comment]
I do apologize for the misunderstanding. I did in fact read through the entire original post, hence why I made this one. But as long as all of what was mentioned in the email is on my post, this shouldn't happen again.
I do admit to missing that the add-ons weren't developed by you. In my development experience, keeping additional content like that separate keep the code cleaner as you dont need to meddle with the "do or do not have that mod" needs for exception catches.
But I do ask, and I know it's hard with some of the posts that are probably made, be a little more cordial. I legitimately saw no further post from admins on the original post other than the "outside scope" part with a tag saying the topic was closed right after. And so was a little hurt when you stated I didnt read the entire thing. On that note though, I'm one person from a sea of nobodies that you not only have to deal with on a day to day but dont care about either. So maybe my point is moot.
Any who, It is a great mod and sorry for the misunderstanding.
Ars Techne    On Sunday, January 20, 2019, 10:23:54 AM MST, Ferreus Veritas <notifications@github.com> wrote:  
 
 
You should have paid attention to the original before opening another issue for the same reason. It's still out of project scope and always will be. This feature will need to be provided as an add-on just like thaumcraft is and will not be done by myself or mangoose.

**Comment by ghost** [comment]
I'm sorry if I came across as rude.  I've had the message about the scope on the curseforge project page for years.  I mentioned this specifically in the FAQ.  I even listed Forestry as an example of what I won't be doing.  I've had to repeatedly say no and explain myself and constantly point out the FAQ.  So when you admitted that you saw the original closed post and decided to submit another issue anyway It's almost as if my opinion of the matter on my project doesn't matter.  Which I perceived as being a bit rude and dismissive.  I can see that's not the kind of person you are and I should have been less terse in my response.

**Comment by ArsTechne** [comment]
Oh, I'm not on forums much and complete forgot there are FAQs. My apologies on that. There was no intent for disrespect, just intrigue as way things were and me being forgetful about forum conduct.
Ars Techne   On Monday, January 21, 2019, 11:24:11 AM MST, Ferreus Veritas <notifications@github.com> wrote:  
 
 
I'm sorry if I came across as rude. I've had the message about the scope on the curseforge project page for years. I mentioned this specifically in the FAQ. I even listed Forestry as an example of what I won't be doing. I've had to repeatedly say no and explain myself and constantly point out the FAQ. So when you admitted that you saw the original closed post and decided to submit another issue anyway It's almost as if my opinion of the matter on my project doesn't matter. Which I perceived as being a bit rude and dismissive.
